app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.where(user_id: current_user.id).first
+    @comment = @post.comments.build(comment_params)
+    @comment.user_id = current_user.id
+    if @comment.valid?
+      @comment.save
+      flash[:notice] = 'Comment created successfully!'
+    else
+      flash[:error] = @comment.errors.full_messages.join(', ')
+    end
+    redirect_to user_post_path(user_id: @post.user_id, id: @post.id)
+  end
+
+  def update; end
+
+  def destroy; end
+
+  private
+
```

Comment:
- [ ] Kindly fix the error which s displayed when I try creating comments on rails c.

<img width="687" alt="Screenshot 2023-01-27 at 2 25 26 PM" src="https://user-images.githubusercontent.com/70269507/215075822-eb4a1ffd-0ee8-4913-8b32-74f0a35b6b79.png">

Reply:
> * [ ]  Kindly fix the error which s displayed when I try creating comments on rails c.
> 
> <img alt="Screenshot 2023-01-27 at 2 25 26 PM" width="687" src="https://user-images.githubusercontent.com/70269507/215075822-eb4a1ffd-0ee8-4913-8b32-74f0a35b6b79.png">

![image](https://user-images.githubusercontent.com/5064641/215080099-5300ab4c-2670-4079-893a-2d289a3bbd66.png)
![image](https://user-images.githubusercontent.com/5064641/215080151-40f9b564-a4fa-4906-b1dd-1d8b296fb190.png)

Reply:
As you can see from my screenshots I can add comments from my end sucessfully.

Reply:
If you dont mind I would like to request a second opinion